utils/get-us-data.R

```diff
@@ -75,10 +75,14 @@ get_us_deaths <- function(data = c("cumulative", "daily")){
 #' 
 
 load_observed_deaths <- function(weekly = FALSE, 
-                                 cumulative = FALSE) {
+                                 cumulative = FALSE, 
+                                 download_newest = FALSE) {
   
   # download newest data
-  get_us_deaths()
+  if (download_newest) {
+    get_us_deaths()
+  }
+  
   
   true_deaths <- (readRDS(here::here("data", "deaths_data.rds"))) %>%
     dplyr::rename(region = state) %>%
```

Comment:
Weekly data needs to be changed over to epiweeks everywhere

Reply:
I added epiweeks to get_us_deaths and get_us_cases. Its use over weeks need to be added in all your code @nikosbosse